src/app/student-login/student-login.component.ts

```diff
@@ -0,0 +1,39 @@
+import { Component, OnInit } from '@angular/core';
+import { AuthService } from '../auth.service';
+import { FirebaseService } from '../firebase.service';
+import { FormControl, FormGroup } from '@angular/forms';
+
+@Component({
+  selector: 'app-student-login',
+  templateUrl: './student-login.component.html',
+  styleUrls: ['./student-login.component.scss']
+})
+export class StudentLoginComponent implements OnInit {
+
+  sessionID: string;
+  nameControl: FormControl;
+  sessionControl: FormControl;
+  sessionFormGroup: FormGroup;
+
+  constructor(public authService: AuthService) {
+    this.sessionID = 'demo-session'; // Temporary until multiple sessions are supported
+  }
+
+  ngOnInit(): void {
+    this.authService.logStudentIn();
+    this.nameControl = new FormControl();
+    this.sessionControl = new FormControl();
+    this.sessionFormGroup = new FormGroup({nameControl: this.nameControl, sessionControl: this.sessionControl});
```

Comment:
This can be done directly when they are declared rather than in `ngOnInit`.

Reply:
Also since you aren't using the individual `FormControl`s anywhere you can just create them right in the constructor for the `FormGroup`. Something like this:
```ts
sessionFormGroup: FormGroup = new FormGroup({
    nameControl: new FormControl(),
    sessionControl: new FormControl(),
  });
```

We should also add some validation to these form inputs. At the very least making both inputs required to click "Join".